packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -77,51 +78,54 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
 
 <SettingsPage title="Extensions">
   <div class="bg-charcoal-600 rounded-md p-3">
-    <div class="bg-charcoal-700 mb-4 rounded-md p-3">
-      <FeaturedExtensions />
-    </div>
-
-    <div class="bg-charcoal-700 mt-5 rounded-md p-3" role="region" aria-label="OCI image installation box">
-      <h1 class="text-lg mb-2">Install a new extension from OCI Image</h1>
-
-      <div class="flex flex-col w-full">
-        <div
-          class="flex flex-row mb-2 w-full space-x-4 items-center"
-          role="region"
-          aria-label="Install Extension From OCI">
-          <Input
-            name="ociImage"
-            id="ociImage"
-            aria-label="OCI Image Name"
-            bind:value="{ociImage}"
-            placeholder="Name of the Image"
-            class="w-1/2"
-            required />
-
-          <Button
-            on:click="{() => installExtensionFromImage()}"
-            disabled="{ociImage === undefined || ociImage.trim() === ''}"
-            class="w-full"
-            inProgress="{installInProgress}"
-            icon="{faArrowCircleDown}">
-            Install extension from the OCI image
-          </Button>
-        </div>
-
-        <div class="container w-full flex-col">
+    <Card title="Featured Extensions">
+      <svelte:fragment slot="content">
+        <FeaturedExtensions />
+      </svelte:fragment>
+    </Card>
+
+    <Card title="Install a new extension from OCI Image" label="OCI image installation box">
+      <svelte:fragment slot="content">
+        <div class="flex flex-col w-full">
```

Comment:
this screen is being changed as part of https://github.com/containers/podman-desktop/issues/2170
so I'm not sure there should be changes for now in that page